web/src/components/molecules/Integration/IntegrationTable/index.tsx

```diff
@@ -3,75 +3,125 @@ import styled from "@emotion/styled";
+      dataIndex: ["integration", "name"],
       key: "name",
       filters: [],
     },
+    {
+      title: t("State"),
+      key: "state",
+      render: (_, integrationMember) => (
+        <Switch
+          checked={integrationMember.active}
+          checkedChildren={t("ON")}
+          unCheckedChildren={t("OFF")}
+        />
+      ),
+    },
     {
       title: t("Role"),
-      dataIndex: "role",
+      dataIndex: "integrationRole",
       key: "role",
     },
     {
       title: t("Creator"),
-      dataIndex: "creator",
+      dataIndex: ["integration", "developer", "name"],
       key: "creator",
     },
     {
-      title: t("State"),
-      dataIndex: "state",
-      key: "state",
-    },
-    {
-      title: t("Action"),
-      dataIndex: "action",
       key: "action",
+      render: (_, integrationMember) => (
+        <Icon
+          style={{ color: "#1890FF", fontSize: "18px" }}
+          onClick={() => onIntegrationSettingsModalOpen(integrationMember)}
+          icon="settings"
+        />
+      ),
     },
   ];
 
   const handleToolbarEvents: ListToolBarProps | undefined = {
     search: {
-      onSearch: _ => {},
+      onSearch: (value: string) => {
+        if (value) {
+          onSearchTerm(value);
+        } else {
+          onSearchTerm();
+        }
+      },
     },
   };
 
   return (
-    <ConfigProvider
-      renderEmpty={() => (
-        <EmptyTableWrapper>
-          <Title>{t("No Integration yet")}</Title>
-          <Suggestion>
-            {t("Create a new")}{" "}
-            <Button onClick={() => {}} type="primary" icon={<Icon icon="api" />}>
-              {t("Connect Integration")}
-            </Button>
-          </Suggestion>
-          <Suggestion>
-            {t("Or read")} <a href="">{t("how to use Re:Earth CMS")}</a> {t("first")}
-          </Suggestion>
-        </EmptyTableWrapper>
-      )}>
-      <ProTable
-        dataSource={dataSource}
-        columns={columns}
-        search={false}
-        rowKey="id"
-        toolbar={handleToolbarEvents}
+    <Wrapper>
+      <PageHeader
+        title={t("Integration")}
+        extra={
+          <Button type="primary" onClick={onIntegrationConnectModalOpen} icon={<Icon icon="api" />}>
+            {t("Connect Integration")}
+          </Button>
+        }
       />
-    </ConfigProvider>
+      <ConfigProvider
+        renderEmpty={() => (
+          <EmptyTableWrapper>
+            <Title>{t("No Integration yet")}</Title>
+            <Suggestion>
+              {t("Create a new")}{" "}
+              <Button
+                onClick={onIntegrationConnectModalOpen}
+                type="primary"
+                icon={<Icon icon="api" />}>
+                {t("Connect Integration")}
+              </Button>
+            </Suggestion>
+            <Suggestion>
+              {t("Or read")} <a href="">{t("how to use Re:Earth CMS")}</a> {t("first")}
+            </Suggestion>
+          </EmptyTableWrapper>
```

Comment:
I guess it was in the other PR that we discussed this being different than the design. Please update here too..

Reply:
@KaWaite , this one is different, the other one was related to adding items, and it didn't have this action.
But for integrations, it has:
![1](https://user-images.githubusercontent.com/78056580/194001107-bcafad81-6bfc-47bb-9d0d-edf33a98ff2f.png)

Reply:
@lavalse Do you want them different like that?

Reply:
So this is fine for now.